hedera-node/hedera-consensus-service-impl/src/test/java/com/hedera/node/app/service/consensus/impl/test/ReadableTopicStoreImplTest.java

```diff
@@ -78,7 +79,7 @@ void getsTopicIfTopicExistsWithNoAutoRenewAccount() {
         assertEquals(adminKey.toString(), topic.submitKey().toString());
         assertEquals(this.topic.sequenceNumber(), topic.sequenceNumber());
         assertEquals(this.topic.autoRenewPeriod(), topic.autoRenewPeriod());
-        assertEquals(0L, topic.autoRenewAccountNumber());
+        assertEquals(0L, topic.autoRenewAccountId().accountNum());
```

Comment:
We should always assert Ids and remove assertions for numbers

Reply:
Updated.